http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/AppControlledResourceComponent.scala

```diff
@@ -88,4 +88,14 @@ object appControlledResourceQuery extends TableQuery(new AppControlledResourceTa
       .filter(_.resourceType === resourceType)
       .result
       .map(_.toList)
+
+  def getAllForAppByResourceId(appId: Long, resourceId: WsmControlledResourceId)(implicit
```

Comment:
Nit, `getAllControlledResourcesForAppByResourceId`?

Reply:
i know it is getting very long, but at least it is explicit, not opinionated wither way  though